lib/gringotts/gateways/monei.ex

```diff
@@ -153,8 +203,8 @@ defmodule Gringotts.Gateways.Monei do
     nil => {nil, nil}
   }
 
-  # MONEI supports payment by card, bank account and even something obscure: virtual account
-  # opts has the auth keys.
+  # MONEI supports payment by card, bank account and even something obscure:
+  # virtual account opts has the auth keys.
```

Comment:
It would be good if your provide a link to the part of Monei documentation which describes this.

Reply:
Link added in the [_real docs_](https://github.com/aviabird/gringotts/blob/iss36-monei-opts/lib/gringotts/gateways/monei.ex#L88).